.github/xml-feed/generate-xml-feed.js

```diff
@@ -0,0 +1,310 @@
+                  ${image}
+                  </SubImage>`;
+    }
+  });
+  const subdivisionXML = `
+  <Subdivision Status="Active">
+    <SubdivisionNumber>${subdivision['SubdivisionNumber']}</SubdivisionNumber>
+    <SubdivisionName>${subdivision['SubdivisionName']}</SubdivisionName>
+    <SubParentName>${subdivision['SubParentName']}</SubParentName>
+    <UseDefaultLeadsEmail>${subdivision['UseDefaultLeadsEmail']}</UseDefaultLeadsEmail>
+    <BuildOnYourLot>${subdivision['BuildOnYourLot']}</BuildOnYourLot>
+    <SalesOffice>
+    <Address OutOfCommunity="0">
+    <Street1>${salesoffice['address']}</Street1>
+    <City>${salesoffice['city']}</City>
+    <State>${salesoffice['zip-code-abbr']}</State>
+    <ZIP>${salesoffice['zipcode']}</ZIP>
+    <Country>USA</Country>
+    <Geocode>
+      <Latitude>${salesoffice['latitude']}</Latitude>
+      <Longitude>${salesoffice['longitude']}</Longitude>
+    </Geocode>
+    </Address>
+    <Phone>
+      <AreaCode>${salesoffice['phone']}</AreaCode>
+      <Prefix>${salesoffice['phone']}</Prefix>
+      <Suffix>${salesoffice['phone']}</Suffix>
+    </Phone>
+    <Hours>${salesoffice['hours']}</Hours>
+    </SalesOffice>
+    <SubAddress>
+    <SubStreet1>${salesoffice['address']}</SubStreet1>
+    <SubCity>${salesoffice['city']}</SubCity>
+    <SubState>${salesoffice['zip-code-abbr']}</SubState>
+    <SubZIP>${salesoffice['zipcode']}</SubZIP>
+    <SubGeocode>
+    <SubLatitude>${salesoffice['latitude']}</SubLatitude>
+    <SubLongitude>${salesoffice['longitude']}</SubLongitude>
+    </SubGeocode>
+    </SubAddress>
+    <DrivingDirections>${salesoffice['DrivingDirections']}</DrivingDirections>
+    <SubDescription>${subdivision['SubDescription']}</SubDescription>
+    ${subImage} 
+    <SubVideoTour Title="Tour of ${subdivision['SubdivisionName']}">${subdivision['SubVideoTour']}</SubVideoTour>
+    <SubWebsite>https://www.hubblehomes.com${subdivision['Path']}</SubWebsite>
+    ${planXML}
+    </Subdivision>`;
+
+  return subdivisionXML;
+}
+
+async function main() {  
+  const dateGenerated = new Date().toJSON();
+  const corporation = await fetchWorkbook([Sheets.CORPORATION]);
+  const filename = 'HubbleHomes_zillow_feed.xml'
+  let corporationXML;
+  corporation.data.forEach((entry) => {
+    corporationXML = `<Corporation>
+    <CorporateBuilderNumber>${entry['CorporateBuilderNumber']}</CorporateBuilderNumber>
+    <CorporateState>${entry['CorporateState']}</CorporateState>
+    <CorporateName>${entry['CorporateName']}</CorporateName>
+      <Builder>
+        <BuilderNumber>${entry['BuilderNumber']}</BuilderNumber>
+        <BrandName>${entry['BrandName']}</BrandName>
+        <ReportingName>${entry['ReportingName']}</ReportingName>
+        <DefaultLeadsEmail LeadsPerMessage="1">${entry['DefaultLeadsEmail']}</DefaultLeadsEmail>
+        <BuilderWebsite>${entry['BuilderWebsite']}</BuilderWebsite>`;
+  });
+  const subdivisionData = await fetchWorkbook([Sheets.SUBDIVISION, Sheets.SALES_OFFICES]);
```

Comment:
What happens if the fetchWorkbook calls fail?

Reply:
if the call fails, we will see an error in the Github workflow console. error will looks like: 
![image](https://github.com/user-attachments/assets/28e4c082-95da-462a-844a-eaef3d55536f)

Reply:
you can also see the trace here: https://github.com/aemsites/hubblehomes-com/actions/runs/10271678159/job/28422206951#step:6:1